website/docs/release_notes.md

```diff
@@ -5,6 +5,167 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.18](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.18)
+
+## Filter tests with `--test-only`
+It is now possible to filter tests with the `--test-only` option.
```

Comment:
Add more precisely, that `--test-only` filter only the tests suite. If someone wants to filter the name of tests to run, he has to pass arguments to framework, for example in `munit`:
```
-- '*foo*'
```

Reply:
should be fixed now